tt_eager/tt_dnn/op_library/composite/composite_ops.cpp

```diff
@@ -2086,11 +2085,11 @@ Tensor _argmax(const Tensor& input_t, int64_t _dim, bool all, const MemoryConfig
                     Tensor tindex = tt::numpy::index_width<bfloat16>(
                         input_shape, DataType::BFLOAT16, Layout::TILE, input_a.device(), output_mem_config);
                     if (is_width) {
-                        max_tensor = bcast(max_tensor, max_val, BcastOpMath::ADD, BcastOpDim::W, output_mem_config);
+                        max_tensor = ttnn::add(max_tensor, max_val, std::nullopt, output_mem_config);
                     } else {
                         tindex = tt::numpy::index_height<bfloat16>(
                             input_shape, DataType::BFLOAT16, Layout::TILE, input_a.device(), output_mem_config);
-                        max_tensor = bcast(max_tensor, max_val, BcastOpMath::ADD, BcastOpDim::H, output_mem_config);
+                        max_tensor = ttnn::add(max_tensor, max_val, std::nullopt, output_mem_config);
```

Comment:
Is the difference between W and H broadcast anyhow reflected in add calls?

Reply:
It's really being reflected by the shape of the tensor going in.  If the dimension is 1 in H then it is broadcasted on the H. (likewise for W).  
Because _dim is an integer it means that will be the case for the correct dimension for max_val